### ISSUE TITLE
JQ command not found

### ISSUE THREAD

**Issue by RyanLiRui** [nunux-keeper/keeper-docker#4]
I can't get the make deploy script to work because the image is missing "jq". I've tried a bunch of ways like changing my path and copying jq into the docker container, but all without luck. Please advise on what I should do here because I really, really would like to use this program.

Log output below.
![support_needed](https://user-images.githubusercontent.com/977609/72662256-f78f9200-3a1f-11ea-82b9-4d9847c7f63e.png)

**Comment by ncarlier** [comment]
Hi,

it's because image version of Keycloak. Using `latest` is not a good idea :/
Can you try to update compose file and use an older version such as `3.4.3.Final` ?

**Comment by RyanLiRui** [comment]
I tried 3.4.3.Final and I still got the same error, so I tried using the version that would have been available when you made your last commit. Unfortunately, I can't get that version, not getting a response from the server when trying to grab 3.0.0.Final. So, then I tried grabbing the next version after that 3.1.0.Final, and I still get the same error as above. Please let me know what the next steps should be.

**Comment by ncarlier** [comment]
Recent version of Keycloak are using Redhat Entreprise as base image.
The packet manager should be `dnf` but only `microdnf` is installed and `jq` is not found.
Therefore the best alternative should be to install `jq` manually.
You can do that by editing [Keycloak Dockerfile](dockerfiles/keycloak/Dockerfile) as follow:

```Dockerfile
FROM jboss/keycloak:latest

USER root

RUN curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 --output /usr/local/bin/jq && chmod +x /usr/local/bin/jq

USER jboss

ADD setup.sh /opt/jboss/setup-keycloak.sh.
```

**Comment by RyanLiRui** [comment]
Thanks for the help ncarlier. I ultimately went back to using 3.4.3.Final after I realised that none of my changes were taking effect, and that I needed to change the tag in a few more places than just the docker-compose.yml file

Firstly, 
"make undeploy"

Then,
"sudo docker system prune -a"
To remove the containers that had been installed but weren't using the right config

Then, change docker-compose.yml (if you haven't already), as instructed to the 3.4.3.Final tag

Then, change docker-compose.config.yml
![docker-compose config yml](https://user-images.githubusercontent.com/977609/72854371-1ecfb300-3cef-11ea-86e1-bbb859171c43.jpg)

Then, change the keycloak Docker file
![dockerfile](https://user-images.githubusercontent.com/977609/72854500-8685fe00-3cef-11ea-822f-d9e1d33d3181.jpg)

And then "make deploy" and it installed properly. 

However, now I'm running into another problem where I can't access the web server. It doesn't appear to be listening. I'll place that in another issue.
